3-extra/1-count-words.js

```diff
@@ -25,11 +25,27 @@
 
 function countWords(string) {
   const wordCount = {};
+  // - Strings and string manipulation 
 
-  // write code here
-
-  return wordCount;
-}
+  // Split the input string into an array of words
+    const words = string.split(" ");
+  
+    // Loop through the array of words
+    words.forEach((word) => {
+     
+      if(string == ""){
+        return wordCount
```

Comment:
there is no need to use `return` when using `forEach` because the return value is discarded anyway (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)